torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -662,8 +667,13 @@ struct to_ir {
   Function& method;
   std::shared_ptr<Graph> graph;
   ResolverPtr resolver;
-  std::unordered_map<int64_t, Value*> integral_constants;
-  std::unordered_map<double, Value*> fp_constants;
+  std::unordered_map<int64_t, Value*, std::hash<int>> integral_constants;
```

Comment:
This looks like a semantic change. Previously, the hasher defaulted to `std::hash<int64_t>`, but you've chopped of some precision here. Was that intentional?

Reply:
good catch ... no it wasn't! I changed that back to `std::hash<int64_t>`

Reply:
you need to do the same with double below